configs/components/recom/recom.yaml

```diff
@@ -190,6 +213,13 @@ restart_out_in_work:
 restart_out_sources:
         bio_restart: fesom.${end_date!syear}.bio.nc
 
+# MA: this needs to go somewhere else ====================
+#4.7.22 Ying: frequency defined in namelist.io, why here?
+yearly_outputs: []
+
+monthly_outputs: [diags3d01,diags3d02,CO2y,aCO2,rDIN,rDON,rDOC,rDSi,PAR,NPPn,NPPd,GPPn,GPPd,NNAn,NNAd,GNAn,GNAd,benCalc,benSi,benC,benN,denb,aN,aFe,Hp,CO2f,dpCO2s,pCO2s,bgc01,bgc02,bgc03,bgc04,bgc05,bgc06,bgc07,bgc08,bgc09,bgc10,bgc11,bgc12,bgc13,bgc14,bgc15,bgc16,bgc17,bgc18,bgc19,bgc20,bgc21,bgc22,slp,age01,Kw,alphaCO2,O2f]
+
+#=========================================================
```

Comment:
This whole file are defaults for REcoM when used with FESOM1 versions. Then, the IO was defined different. We need to take action about this and remove it from here, and put it somewhere in the choose_block above. The question for me is what do we need to put there.

Reply:
Remove this block